sql/exporter.go

```diff
@@ -4,27 +4,81 @@ import (
 	"fmt"
 	"io"
 	"strings"
+	"sort"
 )
 
 const (
-	hierarchySql = "insert into hierarchy (hierarchy_id, hierarchy_name) values ('%s', %s);\n"
-	entrySql     = "insert into hierarchy_entry (hierarchy_id, entry_code, parent_code, code_name, abbreviation, description, level) values ('%s', '%s', %s, %s, %s, %s, %d);\n"
+	hierarchySql = "insert into hierarchy (hierarchy_id, hierarchy_name) values (%s, %s);\n"
+	areaSql = "insert into hierarchy_area_type (id, name, level) select %s, %s, %d where not exists (select id from hierarchy_area_type where id=%[1]s);\n"
```

Comment:
Is the `where not exists` clause to prevent inserting duplicates? Could the same be achieve with `insert into ... values ... on conflict do nothing`?